charts/enterprise-operator/values.yaml

```diff
@@ -41,41 +43,67 @@ operator:
       cpu: 1100m
       memory: 1Gi
 
-  # Create operator-service account
+  # Create operator service account and roles
+  # if false, then templates/operator-roles.yaml is excluded
   createOperatorServiceAccount: true
 
+  # Set to false to NOT create service accounts and roles for the resources managed by the operator
+  # It might be necessary to disable it to avoid conflicts when
+  # kubectl mongodb plugin is used to configure multi-cluster resources
+  createResourcesServiceAccountsAndRoles: true
+
   vaultSecretBackend:
-  # set to true if you want the operator to store secrets in Vault
+    # set to true if you want the operator to store secrets in Vault
     enabled: false
     tlsSecretRef: ''
 
+  # 0 or 1 is supported only
   replicas: 1
+  # additional arguments to pass on the operator's binary arguments, e.g. operator.additionalArguments={--v=9} to dump debug k8s networking to logs
+  additionalArguments: []
+
+  webhook:
+    # registerConfiguration setting (default: true) controls if the operator should automatically register ValidatingWebhookConfiguration and if required for it cluster-wide roles should be installed.
+    #
+    # Setting false:
+    #  - Adds env var MDB_WEBHOOK_REGISTER_CONFIGURATION=false to the operator deployment.
+    #  - ClusterRole and ClusterRoleBinding required to manage ValidatingWebhookConfigurations will not be installed
+    #  - The operator will not create ValidatingWebhookConfigurations upon startup.
+    #  - The operator will not create the service for the webhook. If the `operator-webhook` service was created before, it will be deleted.
+    #  - The operator will still expose the webhook's endpoint on port on MDB_WEBHOOK_PORT (if not specified, the operator uses a default 1993) in case the ValidatingWebhookConfigurations is configured externally (e.g. in OLM/OpenShift) or by the administrator manually.
+    #
+    # Setting true:
+    #  - It's the default setting, behaviour of the operator w.r.t. webhook configuration is the same as before.
+    #  - operator-webhook service will be created by the operator
+    #  - ClusterRole and ClusterRoleBinding required to manage ValidatingWebhookConfigurations will be installed.
+    #  - ValidatingWebhookConfigurations will be managed by the operator (requires cluster permissions)
+    registerConfiguration: true
 
 ## Database
 database:
   name: mongodb-enterprise-database-ubi
-  version: 1.24.0
+  version: 1.25.0
 
 initDatabase:
   name: mongodb-enterprise-init-database-ubi
-  version: 1.24.0
+  version: 1.25.0
 
 ## Ops Manager
 opsManager:
   name: mongodb-enterprise-ops-manager-ubi
 
 initOpsManager:
   name: mongodb-enterprise-init-ops-manager-ubi
-  version: 1.24.0
+  version: 1.25.0
 
 ## Application Database
 initAppDb:
   name: mongodb-enterprise-init-appdb-ubi
-  version: 1.24.0
+  version: 1.25.0
 
 agent:
   name: mongodb-agent-ubi
-  version: 12.0.29.7785-1
+  version: 107.0.0.8502-1
```

Comment:
Just double checking - this seems to be CM Agent and it seems to be the default value. Is that intentional? Shouldn't it correspond to OM 6 agent?

Reply:
Thats an agent for om7, we wanted to be safe and support om6 and om7, thus taking om7 instead of om6